src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/Message.vue

```diff
@@ -12,21 +12,14 @@
           <v-btn prepend-icon="mdi-close" text="Close" @click="messageStore.currentMessage = null"></v-btn>
         </v-toolbar>
         <v-card-text>
-          <v-btn
-            v-if="messageStore.currentMessage?.doNotReply == false"
-            prepend-icon="mdi-reply"
-            variant="text"
-            color="primary"
-            text="Reply"
-            @click="handleMessageReply"
-          >
-            Reply
-          </v-btn>
-          <h2 class="mt-6">{{ messageStore.currentMessage?.subject }}</h2>
+          <v-sheet v-if="messageStore.currentMessage?.doNotReply == false" class="message-reply mb-6">
+            <v-btn prepend-icon="mdi-reply" variant="text" color="primary" text="Reply" @click="handleMessageReply"></v-btn>
+          </v-sheet>
+          <h2>{{ messageStore.currentMessage?.subject }}</h2>
 
           <div v-for="(message, index) in messageStore.currentThread" :key="index" class="small mt-6">
             <span v-html="message.from == 'Registry' ? 'From ECE Registry' : 'PortalUser' ? 'You Replied' : ''"></span>
-            <div class="mt-3" v-html="formatDate(String(message.notifiedOn), 'LLL d, yyyy t')"></div>
+            <div class="mt-3" v-html="`${formatDate(String(message.notifiedOn), 'LLL d, yyyy')} &nbsp; ${formatDate(String(message.notifiedOn), 't')}`"></div>
```

Comment:
do we need $npsp;? 

could 2 spaces be enough width. 

I'm guessing this was done to add an additional space between the time and the date.

Reply:
tried formating with "LLL d, yyyy  t" but that did not work ao added  &nbsp; to introduce additional space between date and time,